samza-core/src/main/java/org/apache/samza/config/BlobStoreConfig.java

```diff
@@ -0,0 +1,81 @@
+package org.apache.samza.config;
+
+import com.google.common.collect.ImmutableList;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+
+/**
+ * Config related helper methods for BlobStore.
+ */
+public class BlobStoreConfig extends MapConfig {
+  private static final String STORE_PREFIX = "stores.";
+  private static final String CHANGELOG_SUFFIX = ".changelog";
+
+  public static final String DEFAULT_BLOB_STORE_STATE_BACKEND_FACTORY =
+      "org.apache.samza.storage.blobstore.BlobStoreStateBackendFactory";
+  public static final String BLOB_STORE_MANAGER_FACTORY = "blob.store.manager.factory";
+  public static final String BLOB_STORE_STATE_BACKEND_ADMIN_FACTORY = "blob.store.state.backend.admin.factory";
+
+  public static final String DEFAULT_STORE_STATE_BACKEND_FACTORY =
+      "org.apache.samza.storage.KafkaChangelogStateBackendFactory";
+  public static final List<String> DEFAULT_STORE_STATE_BACKEND_BACKUP_FACTORIES = ImmutableList.of(
+      DEFAULT_STORE_STATE_BACKEND_FACTORY);
+  public static final String STORE_STATE_BACKEND_BACKUP_FACTORIES = STORE_PREFIX + "%s.state.backend.backup.factories";
```

Comment:
BLOCKER: These are not blob store manager specific, these should be in storage config.

Only the getBlobStoreManagerFactory, getBlobStoreAdminFactory should be here.